vnext/Microsoft.ReactNative/IReactPackageBuilder.idl

```diff
@@ -20,20 +20,23 @@ namespace Microsoft.ReactNative {
 #endif
 
   [webhosthidden]
-  DOC_STRING("A `ReactPackageBuilder` provides the react instance with all the native modules and view managers that will be available in the React instance.")
-  interface IReactPackageBuilder {
-    DOC_STRING("Adds a custom native module. See @ReactModuleProvider.")
+  DOC_STRING("Build ReactNative package with the set of native modules and view managers.")
+  interface IReactPackageBuilder
+  {
+    DOC_STRING("Add a custom native module.  See @ReactModuleProvider.")
     void AddModule(String moduleName, ReactModuleProvider moduleProvider);
 
 #ifndef CORE_ABI
-    DOC_STRING("Adds a custom native module. See @ReactViewManagerProvider.")
+    DOC_STRING("Add a custom native module.  See @ReactViewManagerProvider.")
     void AddViewManager(String viewManagerName, ReactViewManagerProvider viewManagerProvider);
 #endif
   }
 
   [webhosthidden]
-  interface IReactPackageBuilderExperimental
-    requires IReactPackageBuilder {
+  DOC_STRING("Experimental extensions to the @IReactPackageBuilder.")
```

Comment:
this interface should be marked [experimental]

Reply:
I have added the [experimental].